### PR TITLE
Nelshulla

### PR DESCRIPTION
**Pull request by Nelshulla**

### CODE REVIEW

File: lesson_02/quiz/lesson_02_quiz/src/main/java/com/codedifferently/lesson2/Lesson2.java [hunk 1]
```diff
@@ -112,28 +112,28 @@ private static QuizQuestion makeQuestion5() {
...
-        AnswerChoice.UNANSWERED); // Replace `UNANSWERED` with the correct answer.
+        AnswerChoice.A); // Replace `UNANSWERED` with the correct answer.
   }
 
   private static QuizQuestion makeQuestion6() {
     return new QuizQuestion(
         6,
         "What is the decimal number 34 in binary?",
-        "0b00000000"); // Replace the correct digits in the answer.
+        "100010"); // Replace the correct digits in the answer.
```
Comment (anthonydmays) [inline lesson_02/quiz/lesson_02_quiz/src/main/java/com/codedifferently/lesson2/Lesson2.java:122]: The answer is formatted incorrectly. Use the format I've provided for you in the answer.

File: lesson_02/quiz/.vscode/settings.json
Comment (anthonydmays) [inline lesson_02/quiz/.vscode/settings.json:1]: Please revert this and other files. The only modified file should be Lesson2.java

**Review by anthonydmays — CHANGES_REQUESTED** [review]

File: lesson_02/quiz/lesson_02_quiz/src/main/java/com/codedifferently/lesson2/Lesson2.java
Comment (anthonydmays) [inline lesson_02/quiz/lesson_02_quiz/src/main/java/com/codedifferently/lesson2/Lesson2.java:1]: The good news is that the checks pass. The bad news is that you still have 4 more files than you're supposed to have in this PR. Please revert everything but the `Lesson2.java` file and update this pull request.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

File: lesson_02/quiz/lesson_02_quiz/src/main/java/com/codedifferently/lesson2/Lesson2.java
Comment (Nelshulla) [reply lesson_02/quiz/lesson_02_quiz/src/main/java/com/codedifferently/lesson2/Lesson2.java:1]: How do I know what files to revert?

Comment (anthonydmays) [reply lesson_02/quiz/lesson_02_quiz/src/main/java/com/codedifferently/lesson2/Lesson2.java:1]: Anything file not named `Lesson2.java` is one you should revert.

**Comment by anthonydmays** [comment]
Feel free to close this pull request since I've approved the other one.

**Pull request closed without merging**